src/main/java/net/imagej/ops/descriptor3d/AbstractVertexInterpolator.java

```diff
@@ -0,0 +1,35 @@
+package net.imagej.ops.descriptor3d;
+
+
+import org.scijava.ItemIO;
+import org.scijava.plugin.Parameter;
+
+import net.imagej.ops.AbstractComputerOp;
+import net.imagej.ops.AbstractFunctionOp;
+import net.imagej.ops.AbstractOp;
+import net.imagej.ops.Op;
+
+/**
+ * This is the {@link AbstractVertexInterpolator}. A vertex interpolator
+ * computes the real coordinates based on the pixel intensities. 
+ * 
+ * @author Tim-Oliver Buchholz, University of Konstanz
+ *
+ */
+public abstract class AbstractVertexInterpolator extends AbstractOp implements VertexInterpolator {
```

Comment:
No, because I need the real coordinates for the marching cube algorithm. Marching cube tries to approximate the the position of the hull of a point cloud. If we have two pixels of which one is clear on the inside (value == 1) and one is clear on the outside (value == 0) we know that the hull is somewhere in between of these two pixels.